.gitignore

```diff
@@ -20,6 +21,7 @@ output.json
 
 # IntelliJ
 *.iml
+=======
```

Comment:
I forgot to remove this when resolving some conflicts. No biggie though. Should get resolved on next merge.